Lab2_Code.m

```diff
@@ -0,0 +1,204 @@
+% more days by convolving the background with itself N times. We notice how
+% the mean shifts and the amplitude at the mean decreases as we integrate
+% for more days. Also, the distribution is still a Poisson distribution. 
+pdist = makedist("Poisson",'lambda',8);
+pbg = pdf(pdist,0:20);
+
+% Sum over 2 days
+pconv = multiconv(pbg,2);
+px = 0:length(pconv)-1;
+stairs(px, pconv); hold on;
+
+% Sum over 3 days
+pconv = multiconv(pbg,3);
+px = 0:length(pconv)-1;
+stairs(px, pconv); hold on;
+
+% Sum over 4 days
+pconv = multiconv(pbg,4);
+px = 0:length(pconv)-1;
+stairs(px, pconv); hold on;
+
+title("Probability Distribution of Background Summed Over N Days");
+xlabel("Number of Cosmic-Ray Background"); ylabel("Probability");
+legend("N = 2","N = 3","N = 4"); hold off; figure;
+
+%% Section 1B
+% Shows how the summed probability distribution is still a Poisson
+% distribution after 5 days. 
+
+% Sum over 5 days
+pconv = multiconv(pbg,5);
+px = 0:length(pconv)-1;
+stairs(px, pconv); 
+title("Probability Distribution After 5 Days"); 
+xlabel("Number of Cosmic-Ray Background"); ylabel("Probability"); figure;
+
+% From a mathematical perspective, . From a conceptual perspective, .
+
+% CONTINUE SENTENCE
+
+%% Section 1C % INCORRECT
+% Shows how the probability distribution evolves as we average days. The
+% shape of the distribution gets thinner and the discrete features almost
+% smooths out as the number of days become larger. This can be explained by
+% the Central Limit Theorem which states a distribution will approximate a
+% normal distribution as the sample size becomes larger. In relation to
+% Problem 1B, we 
+
+%Also, the
+% amplitude at the mean decreases but the mean doesn't shift when we
+% average days. 
+
+% Average over 2 days
+pconv = multiconv(pbg,2);
+px = 0:length(pconv)-1;
+stairs(px/2, pconv); hold on;
+
+% Average over 5 days
+pconv = multiconv(pbg,5);
+px = 0:length(pconv)-1;
+stairs(px/5, pconv); hold on;
+
+% Average over 10 days
+pconv = multiconv(pbg,10);
+px = 0:length(pconv)-1;
+stairs(px/10, pconv); hold on; 
+
+% Average over 15 days
+pconv = multiconv(pbg,15);
+px = 0:length(pconv)-1;
+stairs(px/15, pconv); hold on;
+
+% Average over 20 days
+pconv = multiconv(pbg,20);
+px = 0:length(pconv)-1;
+stairs(px/20, pconv); hold on;
+
+title("Average Probability Distribution After N Days"); 
+xlabel("Number of Cosmic-Ray Background"); ylabel("Probability"); 
+legend("N = 2","N = 10","N = 15","N = 20"); hold off; figure;
+
+%% Section 1D
+% For this problem, I will choose my number of days as N = 8. According to
+% this problem, I will see Y*N gamma rays which is 128. In order to find
+% the associated sigma to the signal of 128, I must first find the
+% distribution after eight days. This is found by convolving the background
+% with itself 8 times. Afterwards, I must integrate my distribution from
+% negative infinity to the signal to find the associated sigma. 
```

Comment:
I believe it should be 1 minus that integral since we are interested in signals that are bigger than 128 so the right tail of the list. Also you may want to include your statement of the problem